Tests/InstallationTests/CocoapodsInstallation/fastlane/Fastfile

```diff
@@ -0,0 +1 @@
+import("../../../../fastlane/Fastfile")
```

Comment:
Do these installation tests need something from the main `Fastfile`? 🤔

Reply:
🤔 I'm not sure they do anymore!! They did at one point re-use some of the logic from the main Fastfile. 

I'm going to try removing this bit in a separate PR